src/plugins/dashboard/common/dashboard_saved_object/persistable_state/dashboard_saved_object_references.ts

```diff
@@ -59,13 +55,6 @@ export function injectReferences(
     panelsJSON: JSON.stringify(injectedPanels),
   } as DashboardAttributes;
 
-  if (attributes.controlGroupInput && injectedState.controlGroupInput) {
-    newAttributes.controlGroupInput = {
-      ...attributes.controlGroupInput,
-      panelsJSON: JSON.stringify(injectedState.controlGroupInput.panels),
-    };
-  }
```

Comment:
We changed the name of our references, which means that, when a dashboard has been saved with the **new** version of controls, it then cannot be opened on a Dashboard that isn't on the same version:


https://github.com/user-attachments/assets/ece9a781-c3be-46ee-a181-6465c55d77a7


I'm not sure how big of a deal this is - but with serverless, maybe it's safer to just keep the reference names the same? (i.e. switch the references to be the old `controlGroup_<controlId>:optionsListDataView` `controlGroup_<controlId>:rangeSliderDataView` from the current `controlGroup_<controlId>:optionsListControlDataView` `controlGroup_<controlId>:rangeSliderControlDataView`)

Reply:
thanks for pointing this out. Lets keep these the same so there is one less thing to worry about. Resolved in ad2d7bafe79e6efd5ccaa2ccb4d0ac41d928841e